py7zr/compressor.py

```diff
@@ -31,6 +31,7 @@
 
 import pyppmd
 import pyzstd
+from zipfile_deflate64 import deflate64
```

Comment:
I wasn't intending for this to be a public / stable API, so this should have been named `_deflate64`.

However, I'd be happy to help with the usage of zipfile-deflate64 in this project, so I'm willing to make this actually a stake API. I may want to make some small revisions first (in another release). I should be able to do that in the next few days.

Reply:
@brianhelba omg, it is already used in https://github.com/cielavenir/zipfile39/blob/master/zipfile39.py#L77

Reply:
but ok, if minor revision I can follow it